### PR TITLE
feat(src/ru): New source: AnilibriaTV

### LINKED ISSUE #1068: [ru] Anilibria.TV
---

## Source website
- Name: Anilibria.TV
- Link: https://www.anilibria.tv/pages/catalog.php
- Language(s): Russian

## Other details
- API: https://github.com/anilibria/docs/blob/master/api_v2.md

### PR DESCRIPTION
**Pull request by ToxesFoxes**
Closes #1068 
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/ru/anilibriatv/src/eu/kanade/tachiyomi/animeextension/ru/anilibriatv/AniLibriaTV.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package eu.kanade.tachiyomi.animeextension.ru.anilibriatv
+
+import eu.kanade.tachiyomi.animeextension.ru.anilibriatv.api.AniLibriaTVApiV3
+import eu.kanade.tachiyomi.animesource.AnimeSource
+import eu.kanade.tachiyomi.animesource.AnimeSourceFactory
+
+class AniLibriaTV : AnimeSourceFactory {
```
[the remark this answers is not in the document]
Comment (ToxesFoxes) [reply src/ru/anilibriatv/src/eu/kanade/tachiyomi/animeextension/ru/anilibriatv/AniLibriaTV.kt]: Fixed in next commit 👍🏻

**Comment by ToxesFoxes** [comment]
I forgot to fix the main class when I transferred it to a single source, now everything works fine

File: src/ru/anilibriatv/res/mipmap-anydpi-v26/ic_launcher.xml
Comment (jmir1) [inline src/ru/anilibriatv/res/mipmap-anydpi-v26/ic_launcher.xml:1]: Please remove this directory and also remove all *_foreground and *_background and *_monochrome icon files.
They aren't necessary and only create issues on the website.

File: src/ru/anilibriatv/src/eu/kanade/tachiyomi/animeextension/ru/anilibriatv/api_v3.md
Comment (jmir1) [inline src/ru/anilibriatv/src/eu/kanade/tachiyomi/animeextension/ru/anilibriatv/api_v3.md:1]: Is it really necessary to include the entire API documentation?

File: src/ru/anilibriatv/src/eu/kanade/tachiyomi/animeextension/ru/anilibriatv/AniLibriaTV.kt [hunk 2]
```diff
@@ -0,0 +1,241 @@
+package eu.kanade.tachiyomi.animeextension.ru.anilibriatv
+
+import android.app.Application
+import android.content.SharedPreferences
+import android.util.Log
```
Comment (jmir1) [inline src/ru/anilibriatv/src/eu/kanade/tachiyomi/animeextension/ru/anilibriatv/AniLibriaTV.kt:5]: Please remove all logging in your final extension.

**Review by jmir1 — CHANGES_REQUESTED** [review]

**Comment by Secozzi** [comment]
Closing due to inactivity, feel free to open a new pull request later with the requested changes.

**Pull request closed without merging**